File: POO/temas/22/rb/0.rb
# array vacío
frutas = []

# inicializar
frutas = ['piña','coco']

# añade al final
frutas << 'uva'
frutas << 'dátil'
frutas << 'kiwi'
print frutas
puts

# acceso a elementos
puts frutas[0]
puts frutas[4]

# tamaño
puts frutas.length

# modificar un elemento
frutas[2] = 'higo'
print frutas
puts

# slice
print frutas[1..3]
puts
print frutas[3..]
puts
print frutas[..1]
puts
print frutas[-3..-2]
puts

# extrae el último
puts frutas.pop
print frutas
puts

# ordenar no destructivo
print frutas.sort
puts
print frutas
puts

# ordenar destructivo
frutas.sort!
print frutas
puts

# comprobar si está
puts frutas.include?('coco')
puts frutas.include?('manzana')

# eliminar un elemento
frutas.delete_at(2)
print frutas

File: POO/temas/22/py/0.py
# array vacío
frutas = []

# inicializar
frutas = ['piña','coco']

# añade al final
frutas.append( 'uva' )
frutas.append( 'dátil' )
frutas.append( 'kiwi' )
print(frutas)

# acceso a elementos
print( frutas[0] )
print( frutas[4] )
print( frutas[-1] )   # último elemento

# tamaño
print( len(frutas) )

# modificar un elemento
frutas[2] = 'higo'
print(frutas)

# slice
print( frutas[1:4] )
print( frutas[3:] )
print( frutas[:2] )
print( frutas[-3:-1] )

# extrae el último
print( frutas.pop() )
print(frutas)

# ordenar destructivo
frutas.sort()
print(frutas)

# comprobar si está
print( 'coco' in frutas )
print( 'manzana' in frutas )

# eliminar un elemento
del frutas[2]
print(frutas)
